config/schedule.yml

```diff
@@ -1,3 +1,6 @@
+fetch_all_rss:
+  cron: "20 * * * *" # every hour, 20 min after the hour
```

Comment:
Hopefully, this is enough time and won't bring back the SIGTERM trouble we had in the past.

Reply:
I matched what was in the scheduler so I would think so. 
![Screen Shot 2020-08-24 at 3 04 15 PM](https://user-images.githubusercontent.com/1813380/91090705-19d33900-e61b-11ea-9d65-dc226341742c.png)

Reply:
Oh sorry, I misread that, I thought it says every 20 min.